pkg/cmd/server/start/master_args.go

```diff
@@ -74,8 +74,8 @@ func NewDefaultMasterArgs() *MasterArgs {
 		PortalNet:            flagtypes.DefaultIPNet("172.30.17.0/24"),
 		MasterPublicAddr:     flagtypes.Addr{Value: "localhost:8443", DefaultScheme: "https", DefaultPort: 8443, AllowPrefix: true}.Default(),
 		KubernetesPublicAddr: flagtypes.Addr{Value: "localhost:8443", DefaultScheme: "https", DefaultPort: 8443, AllowPrefix: true}.Default(),
-		AssetPublicAddr:      flagtypes.Addr{Value: "localhost:8444", DefaultScheme: "https", DefaultPort: 8444, AllowPrefix: true}.Default(),
-		AssetBindAddr:        flagtypes.Addr{Value: "0.0.0.0:8444", DefaultScheme: "https", DefaultPort: 8444, AllowPrefix: true}.Default(),
+		AssetPublicAddr:      flagtypes.Addr{Value: "https://localhost:8443/console", DefaultScheme: "https", DefaultPort: 8443, AllowPrefix: true}.Default(),
+		AssetBindAddr:        flagtypes.Addr{Value: "0.0.0.0:8443", DefaultScheme: "https", DefaultPort: 8443, AllowPrefix: true}.Default(),
```

Comment:
The only reason these were exposed was so integration tests could pick a free port.